From 626a9f5044e02b0360762c1b4ad24eaf412beefc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=98yvind=20Kol=C3=A5s?= <pippin@gimp.org>
Date: Tue, 23 Aug 2022 12:18:32 +0200
Subject: [PATCH] meson: remove unneded check: true

This check meant to check if we are in a git checkout or a tarball
caused failures when building from tarballs. Hopefully fixing issue #77.
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 8d5fc5eb8..2d38a8a30 100644
--- a/meson.build
+++ b/meson.build
@@ -451,7 +451,6 @@ if git_bin.found() and run_command(
     git_bin,
     'rev-parse',
     '--is-inside-work-tree',
-    check: true,
 ).returncode() == 0
   git_version_h = vcs_tag(
     input : 'git-version.h.in',
-- 
GitLab

